content/applications/finance/fiscal_localizations/italy.rst

```diff
@@ -628,3 +628,83 @@ If the :abbr:`PA (Public Administration)` business doesn't reply within 15 days,
 received deadline notification by email. You can make an arrangement with them and manually set the
 correct :guilabel:`SdI State` on your invoice.
+
+Point of Sale fiscal printers
+=============================
+
+Fiscal regulations mandate using certified RT devices, such as RT printers or RT servers, to ensure
+compliant sales receipts and secure communication with the Tax Authority. These devices
+automatically transmit fiscal data daily. RT printers, designed for individual POS terminals, handle
+transactions, print receipts, and report to the authorities, ensuring data integrity and compliance.
+
+Setting up the printer to work with Odoo
+----------------------------------------
+
+Fiscal printers are meant to work only in the local are network. This means the printer and the
+device running the :doc:`point of sale </applications/sales/point_of_sale>` must be connected to the
+same network.
+Fiscal printers are typically configured to use HTTP by default. To ensure compatibility with Odoo,
+the settings must be updated to enable HTTPS support on the printer.
+
+To set up the printer, either the EpsonFPWizard configuration software or a keyboard connected to
+the printer is required.
+
+.. image:: italy/epson-90-keyboard.png
+   :alt: Fiscal printer keyboard
+
+.. image:: italy/epson-90-screen.png
+   :alt: Fiscal printer screen
+
+To set up the fiscal printer using the keyboard, follow these steps:
+
+#. Make sure that the fiscal printer is set to its default state: printer on, start-up cycle
+   complete, and no transaction in progress.
+#. Type `3333`.
+#. Press :guilabel:`Chiave`. The screen will display :guilabel:`Scleta Funzione`.
+#. Type `34`. The screen will display :guilabel:`Web Server`.
+#. Press :guilabel:`Contante` 3 times until the screen displays :guilabel:`Web Server: SSL`.
+#. To turn the value `0` into a `1`, press :guilabel:`X`.
+#. To confirm, press :guilabel:`Contante` 3 times.
+#. Press :guilabel:`Chiave`.
+
+Then, log in to the printer with the device that runs the :doc:`point of sale
+</applications/sales/point_of_sale>` so it recognizes the printer's certificate.
+
+To approve and install the printer's certificate, follow these steps:
+
+#. To access the printer, open a web browser and enter `https://<ip-of-your-printer>` in the address
+   bar. A :guilabel:`Warning: Potential Security Risk Ahead` security message will then appear.
+#. To show the certificate approval options, click :guilabel:`Advanced`.
+#. To validate the certificate, click :guilabel:`Proceed`.
+
+Then, to ensure the :doc:`point of sale </applications/sales/point_of_sale>` is configured with the
+fiscal printer, go to :menuselection:`Point of Sale --> Configuration --> Settings`. In the
+:guilabel:`Connected Devices` section, add the IP address in the :guilabel:`Italian Fiscal Printer
+IP address` and enable :guilabel:`Use HTTPS`.
+
+Configuring Simulation Mode
+---------------------------
```

Comment:
I think this part should come before the "Setting up the printer to work with Odoo" one to avoid users working in production mode without trying the simulation one.